implementation/contracts/system/TBTCSystem.sol

```diff
@@ -217,6 +308,36 @@ contract TBTCSystem is Ownable, ITBTCSystem, DepositLog {
         return initialCollateralizedPercent;
     }
 
+    /// @notice Get the time remaining until the collateralization thresholds can be updated.
+    function getRemainingCollateralizationUpdateTime() external view returns (uint256) {
+        require(collateralizationThresholdsChangeInitiated > 0, "Update not initiated");
+
+        uint256 elapsed = block.timestamp.sub(collateralizationThresholdsChangeInitiated);
+        return (elapsed >= governanceTimeDelay)?
+        0:
+        governanceTimeDelay.sub(elapsed);
```

Comment:
Can we extract common code to an internal function?

Reply:
sure, i'm ok with that because this isn't a user-facing function so the slightly extra cost doesn't matter much. also deployment should be cheaper